package.json

```diff
@@ -100,39 +102,40 @@
     "fontAwesome": "node_modules/font-awesome-sass/assets/stylesheets"
   },
   "scripts": {
-    "lerna": "lerna",
     "bootstrap": "lerna bootstrap",
     "build": "yarn clean:build && lerna run prebuild && yarn build:babel && yarn build:ts && yarn build:tokens && lerna run postbuild",
     "build:babel": "concurrently \"yarn build:babel:cjs\" \"yarn build:babel:esm\"",
     "build:babel:cjs": "cross-env BABEL_ENV=production:cjs lerna exec --stream --ignore \"$(node ./scripts/getGlobPackages.js babel)\" --  babel src --out-dir dist/js",
     "build:babel:esm": "cross-env BABEL_ENV=production:esm lerna exec --stream --ignore \"$(node ./scripts/getGlobPackages.js babel)\" -- babel src --out-dir dist/esm",
     "build:docs": "yarn build:babel && lerna run postbuild && yarn build:tokens && lerna run build --scope=@patternfly/react-docs --stream",
-    "build:ts": "node ./scripts/copyTS.js",
     "build:tokens": "lerna run build --scope=@patternfly/react-tokens --stream",
+    "build:ts": "node ./scripts/copyTS.js",
     "clean": "yarn clean:build",
     "clean:build": "rm -rf packages/*/dist",
-    "start": "concurrently \"yarn storybook:run\" \"yarn storybook:openurl\"",
     "commit": "git-cz",
+    "coveralls": "cat ./coverage/lcov.info | ./node_modules/coveralls/bin/coveralls.js",
+    "generate": "yarn plop",
```

Comment:
This was the only one added.  I sorted the rest to make it easier to find scripts.